country-dashboard/src/pages/Home.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import currency from "../data/currency-codes.json";
+import language from "../data/language-codes.json";
+import region from "../data/regions.json";
+import countries from "../data/countries.json";
+import SearchBar from "../components/searchBar";
+import SelectOptions from "../components/selectOptions";
+import { DarkModeSwitch } from "react-toggle-dark-mode";
+import "../style/Home.css";
+
+const Home = () => {
+  // Define options for currency and language selection
+  const currencyOptions = Object.keys(currency);
+  const languageOptions = Object.keys(language);
```

Comment:
Language options should use the language name (value) not the code (key)
<img width="359" alt="image" src="https://github.com/rumoravenue/rumor-react-training-a2/assets/62576642/f88d9d80-24c8-44b3-972c-db0f43c59155">